hack/install-libelf.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+VERSION=0.185
+curl -sSfL --retry 5 --retry-delay 3 \
+    "https://sourceware.org/elfutils/ftp/$VERSION/elfutils-$VERSION.tar.bz2" -o- |
+    tar xfj -
+pushd "elfutils-$VERSION"
```

Comment:
This package provide digital signature and sha512 checksum, it would be great to verify the integrity or provenance (preferred) of it before extracting.

https://sourceware.org/elfutils/ftp/0.185/elfutils-0.185.tar.bz2.sig
https://sourceware.org/elfutils/ftp/0.185/sha512.sum

Reply:
Fixed :)